app/app.go

```diff
@@ -117,41 +121,37 @@ var (
 	DefaultNodeHome = os.ExpandEnv("$HOME/.sifnoded")
 
 	ModuleBasics = module.NewBasicManager(
-		append(
-			[]module.AppModuleBasic{
-				genutil.AppModuleBasic{},
-				auth.AppModuleBasic{},
-				bank.AppModuleBasic{},
-				capability.AppModuleBasic{},
-				crisis.AppModuleBasic{},
-				feegrantmodule.AppModuleBasic{},
-				staking.AppModuleBasic{},
-				mint.AppModuleBasic{},
-				distr.AppModuleBasic{},
-				gov.NewAppModuleBasic(
-					paramsclient.ProposalHandler,
-					distrclient.ProposalHandler,
-					upgradeclient.ProposalHandler,
-					ibcclientclient.UpdateClientProposalHandler,
-					ibcclientclient.UpgradeProposalHandler,
-				),
-				params.AppModuleBasic{},
-				upgrade.AppModuleBasic{},
-				slashing.AppModuleBasic{},
-				evidence.AppModuleBasic{},
-				ibc.AppModuleBasic{},
-				ibctransferoverride.AppModuleBasic{},
-
-				clp.AppModuleBasic{},
-				oracle.AppModuleBasic{},
-				ethbridge.AppModuleBasic{},
-				dispensation.AppModuleBasic{},
-				tokenregistry.AppModuleBasic{},
-				admin.AppModuleBasic{},
-				vesting.AppModuleBasic{},
-			},
-			FEATURE_TOGGLE_MARGIN_CLI_ALPHA_getAppModuleBasics()...,
```

Comment:
@Jedi2002 we need to keep this for Margin feature toggle to work

Reply:
Allright , added it back